projects/default/controllers/sumo_supervisor/SumoSupervisor.py

```diff
@@ -445,7 +445,7 @@ def run(self, port, disableTrafficLight, directory, step, rotateWheels,
         self.temporaryDirectory = directory
         self.rootChildren = self.getRoot().getField("children")
         self.viewpointPosition = self.get_viewpoint_position_field()
-        self.maxWebotsVehicleDistanceToLane = 5
+        self.maxWebotsVehicleDistanceToLane = 15
```

Comment:
As for the 2021b, this value was not correctly set to prevent the webotsvehicle from "vanishing" for exemple in the curved road. (when set to 5m the is_on_road test was returning false even if the webotsvehicle was on the road)